Lib/test/test_tarfile.py

```diff
@@ -3593,12 +3657,30 @@ def test_absolute_symlink(self):
         with self.check_context(arc.open(), 'data'):
             self.expect_exception(
                 tarfile.AbsoluteLinkError,
-                "'parent' is a symlink to an absolute path")
+                "'parent' is a link to an absolute path")
+
+    def test_absolute_hardlink(self):
+        # Test hardlink to an absolute path
+        # Inspired by 'dirsymlink' in jwilk/traversal-archives
```

Comment:
Are you talkin about https://github.com/jwilk/traversal-archives ? If yes, I suggest writing the whole URL:

```suggestion
        # Inspired by 'dirsymlink' in https://github.com/jwilk/traversal-archives
```